components/camel-jgroups-raft/src/main/java/org/apache/camel/component/jgroups/raft/JGroupsRaftEndpoint.java

```diff
@@ -0,0 +1,231 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.jgroups.raft;
+
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import org.apache.camel.Component;
+import org.apache.camel.Consumer;
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+import org.apache.camel.Producer;
+import org.apache.camel.component.jgroups.raft.utils.NopStateMachine;
+import org.apache.camel.impl.DefaultEndpoint;
+import org.apache.camel.spi.Metadata;
+import org.apache.camel.spi.UriEndpoint;
+import org.apache.camel.spi.UriParam;
+import org.apache.camel.spi.UriPath;
+import org.jgroups.JChannel;
+import org.jgroups.protocols.raft.StateMachine;
+import org.jgroups.raft.RaftHandle;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The jgroups component provides exchange of messages between Camel and JGroups clusters.
+ */
+@UriEndpoint(firstVersion = "2.23.0", scheme = "jgroups-raft", title = "JGroups raft", syntax = "jgroup-sraft:clusterName", consumerClass = JGroupsRaftConsumer.class, label = "clustering,messaging")
+public class JGroupsRaftEndpoint extends DefaultEndpoint {
+    private static final Logger LOG = LoggerFactory.getLogger(JGroupsRaftEndpoint.class);
+
+    public static final String DEFAULT_JGROUPSRAFT_CONFIG = "raft.xml";
```

Comment:
I would go ..Constants or something else.

Reply:
Probably you may also put all the constants in a constants file